pyspark/test/bigdl/test_simple_integration.py

```diff
@@ -544,9 +545,14 @@ def test_predict(self):
             assert_allclose(p_with_batch[i], ground_label[i], atol=1e-6, rtol=0)
 
         predict_class = model.predict_class(predict_data)
-        predict_labels = predict_class.take(6)
-        for i in range(0, total_length):
-            assert predict_labels[i] == 1
+        if isinstance(predict_class, RDD):
```

Comment:
why we need to check if it's RDD ?  it will only return RDD right ?

Reply:
In local mode, predict_class calls the predictLocalClass API and get a JList[Int] result.
and in distributed mode, predict_class calls the modelPredictClass API and gets a JavaRDD[Sample]. That why I use the if statement here and handle the result separately.